projects/kubernetes/cloud-provider-aws/.gitignore

```diff
@@ -0,0 +1 @@
+cloud-provider-aws
```

Comment:
I believe the output of the credential provider we are building is ecr-credential-provider. Should we not be adding that to .gitignore? Or is this for ignoring the whole directory?